nodes/standalone/src/chain_spec.rs

```diff
@@ -115,43 +105,12 @@ impl Alternative {
 							vec![get_authority_keys_from_secret("//Alice")],
 							get_account_id_from_secret::<ed25519::Public>("//Alice"),
 							vec![
-					// Dev Faucet account
-					get_account_id_from_secret::<ed25519::Public>("receive clutch item involve chaos clutch furnace arrest claw isolate okay together"),
-					get_account_id_from_secret::<ed25519::Public>("//Alice"),
-					get_account_id_from_secret::<ed25519::Public>("//Bob"),
-					get_account_id_from_secret::<sr25519::Public>("//Alice"),
-					get_account_id_from_secret::<sr25519::Public>("//Bob"),
-	],
-						)
-					},
-					vec![],
-					None,
-					None,
-					Some(properties),
```

Comment:
Q: Are the `properties` given to the development spec at another point now?

Reply:
oh sorry, seems to be a confusion because of the diff view here.

Reply:
yeah there was some strange indent going on and i fixed that. But the diff looks strange after that. 😕